gostream/ffmpeg/avcodec/avcodec.go

```diff
@@ -88,14 +95,15 @@ func (ctxt *Context) FreeContext() {
 func (ctxt *Context) SetEncodeParams(width, height int, pxlFmt PixelFormat, hasBFrames bool, gopSize int) {
 	ctxt.width = C.int(width)
 	ctxt.height = C.int(height)
-	ctxt.bit_rate = 2000000
+	ctxt.bit_rate = bitrate
 	ctxt.gop_size = C.int(gopSize)
 	if hasBFrames {
 		ctxt.has_b_frames = 1
 	} else {
 		ctxt.has_b_frames = 0
 	}
 	ctxt.pix_fmt = int32(pxlFmt)
+	ctxt.rc_buffer_size = bitrate * bitrateDeviation
```

Comment:
Added in `rc` value here that allows buffer size to be double the bitrate. This should allow for fluctuations above the target to not impede throughput.